include/numerics/petsc_vector.h

```diff
@@ -1187,7 +1187,7 @@ Real PetscVector<T>::min () const
 
   this->_restore_array();
 
-  PetscErrorCode ierr=0;
+  PetscErrorCode ierr = (PetscErrorCode)0;
```

Comment:
@lindsayad does PETSc `--with-strict-petscerrorcode` warn/error about about C++ style casts, e.g.
```
auto ierr = static_cast<PetscErrorCode>(0);
```
It is a small thing but I prefer to use C++ style casts whenever possible.

Reply:
It works fine. I've opened https://github.com/libMesh/libmesh/pull/3867 to change it

Reply:
OK, thanks, I was going to just update it at some later point, didn't mean to imply that you should do it :)